requirements_testing.txt

```diff
@@ -25,4 +25,5 @@ scikit-learn
 pandas
 qtpy
 pyvista>=0.32
-pyvistaqt>=0.4
\ No newline at end of file
+pyvistaqt>=0.4
```

Comment:
Why do we need `pyvista` and its Qt binding?

Reply:
I was just copying code from MME. is pyvista only needed for 3D plots?

Reply:
Yes exactly, it's only for 3D visualization and we don't yet have 3D features to display ;)

Reply:
Will get rid of it rn